### ISSUE TITLE
Wrong formula for tensorboard step calculation

### ISSUE THREAD

**Issue by brucetusec** [argusswift/YOLOv4-pytorch#17]
@train.py line 149~159
writer.add_scalar('loss_ciou', mloss[0], len(self.train_dataloader) / (cfg.TRAIN["BATCH_SIZE"]) * epoch + i)

The argument step should be 
writer.add_scalar('loss_ciou', mloss[0], len(self.train_dataloader) * epoch + i)

Current formula makes weird result in tensorboard.

![image](https://user-images.githubusercontent.com/22650519/94943134-d4533700-0509-11eb-8426-3341cbd55c8d.png)

**Comment by argusswift** [comment]
> @train.py line 149~159
> writer.add_scalar('loss_ciou', mloss[0], len(self.train_dataloader) / (cfg.TRAIN["BATCH_SIZE"]) * epoch + i)
> 
> The argument step should be
> writer.add_scalar('loss_ciou', mloss[0], len(self.train_dataloader) * epoch + i)
> 
> Current formula makes weird result in tensorboard.
> 
> ![image](https://user-images.githubusercontent.com/22650519/94943134-d4533700-0509-11eb-8426-3341cbd55c8d.png)

The loss function may shake and fail to converge doe to the small batchsize and multi-categories  , especially when the network is complex.You should increase the batchsize as much as possible.
